test

Orlyga01 opened the pull request.

Commit:
test

Commit:
Merge branch 'develop' into admin-office-management

Commit:
[admin-office-manager] added seats support flow

Commit:
[admin-office-manager] SingleChildScrollView support

OzBenDev commented on go_to_office/lib/admin/SeatsList.dart:
```diff
@@ -0,0 +1,101 @@
+            mainAxisSize: MainAxisSize.min,
+             children: <Widget>[
+                 ElevatedButton(
+                child: Text('Add New Seat'),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/SeatsList.dart:
```diff
@@ -0,0 +1,101 @@
+    // Object officateAssreddress = {};
+    return Scaffold(
+        appBar: AppBar(
+          title: Text("Seats"),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/SeatsList.dart:
```diff
@@ -0,0 +1,101 @@
+  Widget build(BuildContext context) {
+
+    // Object officateAssreddress = {};
+    return Scaffold(
```
warp it with  SingleChildScrollView (see login_page.dart)

OzBenDev commented on go_to_office/lib/admin/office.page.dart:
```diff
@@ -0,0 +1,96 @@
+    // Object officateAssreddress = {};
+    return Scaffold(
+        appBar: AppBar(
+          title: Text("Office Manager"),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/office.page.dart:
```diff
@@ -0,0 +1,96 @@
+                TextFormField(
+                  initialValue: this.officeDescription,
+                    decoration: InputDecoration(
+                        hintText: 'Please insert Office Description'),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/office.page.dart:
```diff
@@ -0,0 +1,96 @@
+                TextFormField(
+                  initialValue: this.officeCoutnry,
+                    decoration:
+                        InputDecoration(hintText: 'Please insert Country'),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/office.page.dart:
```diff
@@ -0,0 +1,96 @@
+                TextFormField(
+                  initialValue: this.officeName,
+                    decoration:
+                        InputDecoration(hintText: 'Please insert Office name'),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/office.page.dart:
```diff
@@ -0,0 +1,96 @@
+                      });
+                    }),
+                    ElevatedButton(
+                      child: Text('Add New Seat'),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/officesList.dart:
```diff
@@ -0,0 +1,85 @@
+            mainAxisSize: MainAxisSize.min,
+             children: <Widget>[
+              ElevatedButton(
+                child: Text('Add New Office'),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/seat.page.dart:
```diff
@@ -0,0 +1,57 @@
+                  TextFormField(
+                  initialValue: this.seatLocation,
+                    decoration: InputDecoration(
+                        hintText: 'Please insert Seat Loaction in the room'),
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/admin/seat.page.dart:
```diff
@@ -0,0 +1,57 @@
+  Widget build(BuildContext context) {
+    this.roomId = '1';
+    if (this.seatId != null) {  // call API to get get office also get the office name
+    this.seatLocation = 'Near a window';
```
Add to the Strings and import it

OzBenDev commented on go_to_office/lib/login_page.dart:
```diff
@@ -32,13 +33,13 @@ class _LoginPageState extends State<LoginPage> {
   Widget build(BuildContext context) {
     void showSnack(String text) {
       final snackBar = SnackBar(content: Text(text));
```
remove this line also - make this method empty for now

OzBenDev commented on go_to_office/lib/login_page.dart:
```diff
@@ -32,13 +33,13 @@ class _LoginPageState extends State<LoginPage> {
 
     void onError(String text) {
       final snackBar =
           SnackBar(content: Text(text), backgroundColor: Colors.red);
```
remove this line also - make this method empty for now

OzBenDev commented on go_to_office/lib/admin/officesList.dart:
```diff
@@ -0,0 +1,85 @@
+                    child: Container(
+                      height: 50,
+                      margin: EdgeInsets.all(2),
+                      child: Text('${officesList[index].name}',
```
null check? can it be problematic?

OzBenDev commented on go_to_office/lib/admin/officesList.dart:
```diff
@@ -0,0 +1,85 @@
+                itemCount: officesList.length,
+                itemBuilder: (BuildContext context, int index) {
+                  return new GestureDetector(
+                    onTap: () => openOfficePage(officesList[index].id),
```
null check? can it be problematic?

Orlyga01 replied:
If the officesList is empty the loop will not run

OzBenDev commented on go_to_office/lib/admin/SeatsList.dart:
```diff
@@ -0,0 +1,101 @@
+
+void main(officeId) {
+  runApp(new MaterialApp(
+    title: "Seats",
```
Add to the Strings and import it

OzBenDev reviewed (changes requested):
Add null checks if needed, use the modify and import the Strings resource file

Commit:
Co
Merge branch 'develop' into admin-office-management

# Conflicts:
#	go_to_office/lib/login_page.dart

Commit:
Merge branch 'develop' into admin-office-management

# Conflicts:
#	go_to_office/lib/login_page.dart

Commit:
update login page

Commit:
Merge branch 'develop' into admin-office-management

Commit:
Added seats and office model in seperated files
